Got error message about signature in the core log when deleting artifacts

This shouldn't be logged as error:
```
Apr  8 07:15:12 192.168.112.1 core[45846]: 2020-04-08T07:15:12Z [ERROR] [/pkg/signature/notary/helper.go:93]: Repository not exist, repo: 10.193.59.96/yinw/registry, error: notary-server:4443 does not have trust data for 10.193.59.96/yinw/registry, returning empty signature
```

--- a/src/pkg/signature/notary/helper.go
+++ b/src/pkg/signature/notary/helper.go
@@ -90,7 +90,7 @@ func GetTargets(notaryEndpoint string, username string, fqRepo string) ([]model2
 	}
 	targets, err := notaryRepo.ListTargets(data.CanonicalTargetsRole)
 	if _, ok := err.(client.ErrRepositoryNotExist); ok {
-		log.Errorf("Repository not exist, repo: %s, error: %v, returning empty signature", fqRepo, err)
+		log.Debugf("Repository not exist, repo: %s, error: %v, returning empty signature", fqRepo, err)
 		return res, nil
 	} else if err != nil {
 		return res, err

